apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -355,6 +356,18 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
             }
         }
 
+        // If the selected template is M2M
```

Comment:
Shall we remove this comment as the code is self-explanatory?

Reply:
Fixed with 4c374e298f1181dbe40bbbfb9ed9fdae8ef2bb8b